content/finite-fields/en/finite-fields.md

```diff
@@ -175,7 +178,7 @@ Some examples:
 
 The advantage of this approach is we can use the `expmod` [precompile in Ethereum](https://www.rareskills.io/post/solidity-precompiles) to compute the modular inverse in a smart contract.
 
-In practice, this is not an ideal way to compute multiplicative inverses because raising a number to a large power is computationally expensive. Libraries that compute the multiplicative inverse use more efficient algorithms under the hood. However, when such a library is not a available, Fermat’s Little Theorem can be used.
+In practice, this is not an ideal way to compute multiplicative inverses because raising a number to a large power is computationally expensive. Libraries that compute the multiplicative inverse use more efficient algorithms under the hood. However, when such a library is not a available, or if you want to calculate the inverse yourself in a small finite field, Fermat’s Little Theorem can be used.
```

Comment:
even in a large finite field, this could be used depending on the implementation of the library / language.

Maybe "However, when such a library is not a available, and you want a quick and simple solution, and computing a large exponent is not excessively costly, Fermat's Little Theorem can be used.